### PR TITLE
feat: improve address format utils

### PR DESCRIPTION
**Pull request by DennyWanye**
process fuction for address format

### CODE REVIEW

File: packages/eth-explorer/src/buttons/TransferButton.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ export default class TransferButton extends PureComponent {
     if (!from || !await networkManager.sdk.isValidAddress(from)) {
       return
     }
+
```
Comment (iNorthIsle) [inline packages/eth-explorer/src/buttons/TransferButton.js:56]: please remote the line

File: packages/eth-sdk/src/utils/index.js [hunk 2]
```diff
@@ -60,8 +60,25 @@ function parseValue (value, param, chainId) {
 
 export default {
   txOptions,
-  isValidAddress: address => ethers.utils.isAddress(address),
-  formatAddress: address => ethers.utils.getAddress(address),
+  isValidAddress: address => {
```
Comment (iNorthIsle) [inline packages/eth-sdk/src/utils/index.js:63]: should return true or false

Comment (DennyWanye) [reply packages/eth-sdk/src/utils/index.js]: test for start a review

**Review by DennyWanye — COMMENTED** [review]
test

File: packages/eth-explorer/src/buttons/TransferButton.js (hunk 1 above)
Comment (DennyWanye) [reply packages/eth-explorer/src/buttons/TransferButton.js:56]: the line is removed.

File: packages/eth-sdk/src/utils/index.js [hunk 3]
```diff
@@ -61,7 +61,16 @@ function parseValue (value, param, chainId) {
 export default {
   txOptions,
   isValidAddress: address => ethers.utils.isAddress(address),
-  formatAddress: address => ethers.utils.getAddress(address),
+  formatAddress: address => {
+    if(!ethers.utils.isAddress(address)) {
```
Comment (iNorthIsle) [inline packages/eth-sdk/src/utils/index.js:65]:
```suggestion
    return this.isValidAddress(address) : this.formatAddress(address) : '---'
```

**Review by iNorthIsle — APPROVED** [review]

**Pull request merged**